packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -207,6 +210,14 @@ export const useCustomValidity = (options: UseCustomValidityOptions) => {
     };
   });
 
+  const successMessages = computed<FormMessages | undefined>(() => {
+    if (validityState.value === undefined || !validityState.value.valid) return;
+
+    const customMessage = getFormMessages(options.props.customMessage);
+
+    return customMessage;
+  });
```

Comment:
```suggestion
  const successMessages = computed<FormMessages | undefined>(() => {
    if (validityState.value === undefined || !validityState.value.valid) return;
    return getFormMessages(options.props.customMessage);
  });
```

We can make this more compact :)